src/components/Checkbox/index.js

```diff
@@ -29,69 +31,69 @@ export default class Checkbox extends React.Component {
     disabled: PropTypes.bool,
     checked: PropTypes.bool,
     onChange: PropTypes.func,
+    reset: PropTypes.bool,
   }
 
   static defaultProps = {
     className: '',
     inputClassName: '',
     style: {},
+    light: false,
+    checked: false,
     onChange: () => {},
+    reset: false,
   }
 
-  state = {
```

Comment:
Internal state was there to preserve the uncontrolled input behaviour, @accurat/developers do you think it makes sense to keep it? maybe it's really not necessary..

Reply:
This way the `checked` status would only derive from prop

Reply:
I'm not really sure, do we want to use it as an uncontrolled component? If we do this, we have to ensure consistency in every other input component and enforce they are controlled.

Reply:
I think that enforcing controlled inputs could prevent a lot of possible errors.

Reply:
agreed, added issue to update other input components as well: #12